starter-code/functions-and-arrays.js

```diff
@@ -1,14 +1,28 @@
 // Find the maximum
 function maxOfTwoNumbers (first, second) {
-
+  if (first >= second) {
+    return first
+  } else {
+    return second
+  }
 }
 
 var largest = maxOfTwoNumbers(2, 6);
 console.log(largest);
 
+
 // Finding Longest Word
 function findLongestWord (words) {
+  var max_length = 0
```

Comment:
This is a very good solution. However you don't need `max_length` because you can always get it from `max_word.length`.